packages/suite/src/types/suite/guide.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+export type GuideView = 'GUIDE_DEFAULT' | 'GUIDE_CATEGORY' | 'GUIDE_ARTICLE';
+export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBACK_SUGGESTION';
+export type ActiveView = GuideView | FeedbackView;
+export type Category =
+    | 'dashboard'
+    | 'account'
+    | 'settings'
+    | 'send'
+    | 'receive'
+    | 'trade'
+    | 'other';
+export type Rating = 1 | 2 | 3 | 4 | 5;
+export type FeedbackType = 'BUG' | 'SUGGESTION';
+export type FeedbackPayloadBug = {
+    type: FeedbackType;
```

Comment:
Since this is inside `FeedbackPayloadBug` should it rather be
```suggestion
    type: 'BUG';
```
?